src/main/java/net/mcreator/element/types/LivingEntity.java

```diff
@@ -60,8 +62,8 @@
 	public String mobLabel;
 
 	@Nonnull public String mobModelName;
-	public String mobModelTexture;
-	public String mobModelGlowTexture;
+	@TextureReference(TextureType.ENTITY) public String mobModelTexture;

```

Comment:
If nothing else, this ME also defines guiBoundTo, which is ME reference. Please recheck GEs for missing references like this one

Reply:
:heavy_check_mark: Fixed